src/main/scala/org/broadinstitute/dsde/vault/DmClientService.scala

```diff
@@ -14,15 +14,18 @@ import org.broadinstitute.dsde.vault.services.ClientFailure
 import spray.client.pipelining._
 import spray.http.HttpHeaders.Cookie
 import spray.httpx.SprayJsonSupport._
+import spray.json.DefaultJsonProtocol
 import spray.routing.RequestContext
 
 import scala.util.{Failure, Success}
 
+
 object DmClientService {
   case class DMCreateUBam(ubam: UBamIngest)
   case class DMUBamCreated(createdUBam: UBam)
   case class DMResolveUBam(ubamId: String)
   case class DMUBamResolved(dmObject: UBam)
+  case class DMResolveUBamList()
```

Comment:
Generally, I assume one would want a singleton, versus creating / GC'ing multiple empty class instances.

The singleton would be `case object DMResolveUBamList`.